wiki/markdown/compatibility.md

```diff
@@ -1,10 +1,10 @@
 # Compatibility
 
-|                      |Front-end ➡ |         |         |         |
-|---------------------:|:----------:|:-------:|:-------:|:-------:|
-|**Back-end ⬇**        |       1.0.0|    1.0.1|    1.0.2|    1.0.3|
-|[1.0.0](#back-end-100)|[✔️️](#front-end-100)  |[✔️️](#front-end-101)    |[✅](#front-end-102)  |[✅](#front-end-103)  |
-|[1.0.1](#back-end-101)|[✔️️](#front-end-100-1)|[✔️️](#front-end-101-1)  |[✅](#front-end-102-1)|[✅](#front-end-103-1)|
+|                      |Front-end ➡           |                      |                      |                      |
+|---------------------:|:--------------------:|:--------------------:|:--------------------:|:--------------------:|
+|**Back-end ⬇**        |                 1.0.0|                 1.0.1|                 1.0.2|                 1.0.3|
+|[1.0.0](#back-end-100)|[✔️️](#front-end-100)   |[✔️️](#front-end-101)   |[✅](#front-end-102)  |[✅](#front-end-103)  |
+|[1.0.1](#back-end-101)|[✔️️](#front-end-100-1) |[✔️️](#front-end-101-1) |[✅](#front-end-102-1)|[✅](#front-end-103-1)|
 |[1.0.2](#back-end-102)|[❕](#front-end-100-2)|[✅](#front-end-101-2)|[✔️️](#front-end-102-2) |[✔️️](#front-end-103-2) |
```

Comment:
I want an introductory paragraph stating our recommendations about always using the latest versions whenever possible :)

Reply:
Done. I wrote something down.